docs/tables.md

```diff
@@ -186,4 +186,58 @@ Reduced Scans
+| write       | Writing to the remote. For non-`HTTPS` tests this is where the domain is sent. |
+| read        | Reading from the remote |
+| http        | Verification of HTTP headers. `HTTP/S` only |
+| content     | Verification that the returned content matches the expected content. These are the most common types of errors and represent things like blockpages. |
+| final       | Reaching the final stage without encountering any problems in the previous stages. |
+| unknown     | The stage of the error is not known. Usually these are new errors which should be investigated and classified  |
+
+##### Error Class
+
+Protocol errors are mostly defined by the Network Error Logging standard's [Predefined Network Error Types](https://www.w3.org/TR/network-error-logging/#predefined-network-error-types). We add one extended error type, `tcp.no_route`. These errors may represent normal network failures and noise, but they may also expose packets inserted by a middlebox to disrupt a connection. For example China and Iran are both known to use [TCP Reset Attacks](https://en.wikipedia.org/wiki/TCP_reset_attack) as a censorship method.
+
+Mismatch Errors are used when the connection is successful, but the content received does not match the content expected. This can happen in the case of blockpages, remote servers with unusual behavior, or complicated CDN networks serving many sites.
+
+| Error Class             | Explanation |
+| ----------------------- | ----------- |
+| success                 | The test completed successfully and no interference was detected |
+| invalid                 | The test was invalid, usually a test system failure |
+| unknown                 | The class of the error is not known. Usually these are new errors which should be investigated and classified |
+|                         | |
+| **Protocol Errors**     | [Predefined Network Error Types](https://www.w3.org/TR/network-error-logging/#predefined-network-error-types) |
+|                         | |
+| tcp.address_unreachable | The IP address is unreachable |
+| tcp.no_route            | No route to the IP address could be found |
+| tcp.refused             | The TCP connection was refused by the server |
+| tcp.closed              | The TCP connection was closed by the server. Could indicate an inserted `FIN` packet |
+| tcp.reset               | The TCP connection was reset. Could indicate an inserted `RST` packet |
+| tcp.timed_out           | The TCP connection to the server timed out. Could indicate packets being dropped by a middlebox |
+| tls.protocol.error      | The TLS connection was aborted due to a TLS protocol error |
+| tls.failed              | The TLS connection failed due to reasons not covered by other errors |
+| http.response.invalid   | Response is empty, has a content-length mismatch, has improper encoding, and/or other conditions that prevent user agent from processing the response |
```

Comment:
Let's have one dot only. Perhaps http.invalid or http.malformed. I think we can mention that it's an invalid HTTP response (doesn't respect the protocol).

Reply:
used http.invalid